src/main/configurations/xxllnc/Configuration_Handler_AddDocumentToCaseCommand.xml

```diff
@@ -105,27 +105,66 @@
                 </IbisLocalSender>
                 <Param name="Alpha3" sessionKey="ZgwEnkelvoudigInformatieObject" xpathExpression="//taal" />
                 <Forward name="success" path="EnrichZgwEnkelvoudigInformatieObject" />
-            </SenderPipe>
+            </SenderPipe> -->
 
             <XsltPipe
                 name="EnrichZgwEnkelvoudigInformatieObject"
```

Comment:
The enrich should not be needed if you are not overwriting "taal". sessionkey "ZgwEnkelvoudigInformatieObject" you already have